### ISSUE TITLE
Possible error in branchdeducing_twofinite

### ISSUE THREAD

**Issue by pmacmill** [WMiller256/Alhazen-Ptolemy#2]
In your paper, the E6 coefficient denominator is 16*E5, but in the python code it is 4*E5. Similarly in the calculation of E7, the first 2 terms are different from the paper, with the code using b2*c_obs2/2 - E0/3 and in the paper this is 3*(b*cos theta_obs)^2 / 12 - 2*E0/12.

Am I missing something in the algebra? E4 in the code matches your paper.

**Comment by WMiller256** [comment]
Thank you for bringing this to my attention. 

The implementation is correct, it has been thoroughly validated against multiple numerical models (one of which is included in `alhazenptolemy.py`). It appears the equations in the paper are wrong -- I suspect I made an algebraic modification to save a few FLOPs in the implementation and failed to propagate it through all the equations in the text. I will notify the journal.

In the meantime, I have increased the maximum iteration limit for the `numerical` function and added a call to it in the `main` execution for `alhazenptolemy.py`, running the routine from the command line will now print both the analytical and numerical result.
